### PR TITLE
simplify the integration of the state estimation

### PR DESCRIPTION
**Pull request by mstoelzle**
The following steps need to be taken to use the state estimation:

1. Initialisation and calibration for ground temperature and pressure:
```
state_est_state_t state_est_state = { 0 };
reset_state_est_state(ground_pressure, ground_temperature, &state_est_state);
```
2. Update of state estimation measurements
```
state_est_state->state_est_meas = xy;
```
3. Update of state estimation measurements (with included flight detection step)
```
state_est_step(tick_count, &state_est_state, true);
```

### CODE REVIEW

File: Inc/Util/util.h [hunk 1]
```diff
@@ -8,12 +8,7 @@
 #ifndef INC_UTIL_UTIL_H_
 #define INC_UTIL_UTIL_H_
 
-//#include "cmsis_os.h"
-
-/* as we don't include cmsis_os.h, we need to define some datatypes ourselves */
-typedef u_int8_t uint8_t;
-typedef u_int16_t uint16_t;
-typedef u_int32_t uint32_t;
+#include "cmsis_os.h"
```
Comment (hacknus) [inline Inc/Util/util.h:11]: cmsis_os is also a freeRTOS library, not needed for recovery

Comment (mstoelzle) [reply Inc/Util/util.h:11]: Yes, I put everything related to freeRTOS into the `util.h`. Recovery will not need to include this file, as there are not any dependencies in the code

**Review by hacknus — APPROVED** [review]
nice! will try to implement (recovery) in the coming days

**Review by jbinz — APPROVED** [review]
I like the update. Can be mergen IMO.

**Pull request merged**